source/cgame/cg_public.h

```diff
@@ -367,6 +369,9 @@ typedef struct
 
 
 	void ( *CallbackRequestAvatar )( uint64_t steamid, char *avatar );
+
+
+	blockentry_t (* blockentries) [MAX_BLOCKS];
```

Comment:
Directly exporting data like this is not an ideal interface,
because all your code is now starting to depend on an implementation detail.
If you ever want to change the blocklist, you suddenly have to edit 3 or 4 modules.

Reply:
What would be more idiomatic? Having an entire export function for enumeration, lookup and length seems unnecessary

Reply:
Most enumeration functions (maplist, files, ...) just keep enumerating until there are no results left, so they don't need a count.
Lookup could be a single import, and adding/removing is already done with commands.